apps/fluent-tester/src/TestComponents/Text/TextTest.tsx

```diff
@@ -3,13 +3,18 @@ import { StandardUsage } from './StandardUsage';
 import { CustomizeUsage } from './CustomizeUsage';
 import { PressableUsage } from './PressableUsage';
 import { E2ETextTest } from './TextE2ETest';
+import { V2Usage } from './V2Usage';
 import { Test, TestSection, PlatformStatus } from '../Test';
 
 const textSections: TestSection[] = [
   {
     name: 'Standard Usage',
     component: StandardUsage,
   },
+  {
+    name: 'V2 Usage',
+    component: V2Usage,
```

Comment:
We're on V2 now? Not V1?

Reply:
I think we already have a test for this in the Experimental Text Test...

Sorry for the confusion, but the experimental text test tests the newer Text version. We haven't combined the Text test pages yet.

Reply:
Done.